geometries/make_structured_mesh.py

```diff
@@ -38,6 +38,14 @@ def _get_variant_geo_file(variant: str) -> str:
     return f"spe11{variant.lower()}.geo"
 
 
+def _get_domain_dimensions(variant: str) -> tuple:
+    return {
+        "A": ((0.0, 0.0, 0.0), (2.8, 1.2, 0.0)),
+        "B": ((0.0, 0.0, 0.0), (8400, 1200.0, 0.0)),
+        "C": ((0.0, 0.0, 0.0), (8400.0, 5000.0, 1354.0))
```

Comment:
I believe evaluating the given expression yields a different upper bound for the z-coordinate:

```suggestion
        "C": ((0.0, 0.0, 0.0), (8400.0, 5000.0, 1355.0))
```